source/world_builder/features/subducting_plate_models/temperature/mass_conserving.cc

```diff
@@ -312,8 +345,7 @@ namespace WorldBuilder
                 }
 
               // Plate age increases with distance along the slab in the mantle
-              double effective_plate_age = plate_age_sec + (distance_along_plane / plate_velocity) * seconds_in_year; // m/(m/y) = y(seconds_in_year)
-
+              double effective_plate_age = plate_age_sec + (distance_along_plane / subducting_velocity + effective_age_shift) * seconds_in_year; // m/(m/y) = y(seconds_in_year)
```

Comment:
Isn't that just the plate_age_sec + (distance_along_plane / subducting_velocity)? So the difference to the plate_age_sec is that the material is subducted so an age is added?

Reply:
I renamed `plate_velocity` to `subducting_velocity` so it was clearer to me which velocity was which while writing this code, but `effective_age_shift` is an additive age yes. The difference between `distance_along_plane / subducting_velocity` and `effective_age_shift` is that `effective_age_shift` only applies over a certain length of the slab

Reply:
Ahh yes you're definitely right. I need to be applying that to the unsubducted oceanic plate so that the ages match up, not the subducting plate!